src/modules/auth/resources/ResourcesDetailContainer.js

```diff
@@ -78,22 +108,36 @@ const ResourceDetailContainer = (props) =>  {
 							</CardHeader>
 
 							<CardBody>
-								<Row className="card-body-row">
+								<Row>
 									<Col md={3}>{t("Name")}</Col>
-									<Col>{resource._id}</Col>
+									<Col>
+										{ editMode ?
+											<Input
+												id="resource_name"
+												name="resource_name"
+												type="text"
+												autoComplete="off"
+												onChange={registerName.onChange}
+												onBlur={registerName.onBlur}
+												innerRef={registerName.ref}
+											/>
+										:
+											resource && resource._id
```

Comment:
@petrKavulok you can use conditional rendering here (and elsewhere), it will have the same result

```
resource?._id
```